etc/kayobe/globals.yml

```diff
@@ -64,6 +64,13 @@
 # to not specify a gather subset.
 #kayobe_ansible_setup_gather_subset:
 
+###############################################################################
+# To work around issue of trying to install docker from
+# empty pulp server, use upstream docker dnf repo on
+# non-overcloud hosts
+enable_docker_repo: "{{ 'overcloud' not in group_names or ansible_facts.os_family == 'Debian' }}"
```

Comment:
The default is `true`, so we could set to `false` in overcloud group_vars?

Not sure about the Debian bit here, the Docker repo is in apt.yml so we should use the Pulp repo rather than upstream.

Reply:
I've just moved this over from `kolla/globals.yml`. again from [this](https://docs.openstack.org/releasenotes/kayobe/2023.1.html#:~:text=*%20%60%60enable_docker_repo%60%60) release note

Reply:
Moved it over to group_vars